bin/oneoff/backfill_data/user_scripts_for_user_levels

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env ruby
+
+# This script creates a UserScript for every row in the input CSV file.
+#
+# This script will be run with a CSV containing the user IDs, script IDs, and associated timestamps
+# for the approximately ZZZK production UserScripts that remain to be backfilled by
+# User#backfill_user_scripts. As there other instances of a UserLevel (with a script_id) existing
+# without the corresponding UserScript existing, we "fix" them here as well.
+#
+# This CSV will be created via SQL run on AWS Redshift, as the reporting database is not performant
+# enough.
+#
+#   SELECT DISTINCT
+#     users.id AS user_id,
+#     user_levels.script_id,
+#     MIN(user_levels.created_at) AS started_at,
+#     MAX(user_levels.updated_at) AS last_progress_at,
+#     COUNT(0) AS num_levels
+#   FROM users
+#   INNER JOIN user_levels
+#     ON user_levels.user_id = users.id
+#   LEFT OUTER JOIN user_scripts
+#     ON user_scripts.user_id = users.id
+#       AND user_scripts.script_id = user_levels.script_id
+#   WHERE user_scripts.id IS NULL
+#   GROUP BY 1,2;
+#
+# After this script is run, User#backfill_user_scripts will be removed.
+
+require 'csv'
+require_relative '../../../dashboard/config/environment'
+
+NUM_LEVELS_REQUIRED = {
+  # A hash mapping a script_id to the number of levels required for "completion".
+  1 => 100,
+  2 => 1,
+  18 => 100
+}.freeze
```

Comment:
Note that this hash still needs to be populated. However, I want to merge this so that I can play around with it on  environments (e.g., `staging`) with more data than I have locally.